src/sentry/workflow_engine/models/workflow_fire_history.py

```diff
@@ -12,6 +14,7 @@ class WorkflowFireHistory(DefaultFieldsModel):
     group = FlexibleForeignKey("sentry.Group", db_constraint=False)
     event_id = CharField(max_length=32)
     notification_uuid = UUIDField(auto_add=True, unique=True)
+    has_fired_actions = models.BooleanField(default=False)  # whether or not actions were triggered
```

Comment:
Remind me how this works again? What cases do we have where this will be false when we create the fire history? Do we just set it to true if any action fired?

Reply:
https://github.com/getsentry/sentry/blob/397ed78d84dfb43283bc5746892666eb67dda924/src/sentry/workflow_engine/processors/workflow.py#L230
^ we will create this row for triggered workflows here

https://github.com/getsentry/sentry/blob/397ed78d84dfb43283bc5746892666eb67dda924/src/sentry/workflow_engine/processors/workflow.py#L126-L129
^ if we evaluate the filter conditions and find that we should fire actions, we would update this column to `True`. it's possible that the filter conditions are not met and we do not fire actions, in which case we would not update the column